controllers/admin/AdminImportController.php

```diff
@@ -2709,8 +2710,8 @@ protected function productImportOne($info, $idDefaultLanguage, $idLang, $forceId
             $product->id_category = array_values(array_unique($product->id_category));
         }
 
-        // Will update default category if there is none set here. Home if no category at all.
-        if (!isset($product->id_category_default) || !$product->id_category_default) {
+        // Will update default category if forced or if there is none set here. Home if no category at all.
+        if ($forceCat || !isset($product->id_category_default) || !$product->id_category_default) {
```

Comment:
Variable `$forceCat` is undefined here. Did you forget to pass it to `productImportOne` method?

Reply:
https://github.com/thirtybees/thirtybees/pull/1060/commits/f760274967338fcd49fcc2be6e8bc8c1ee4d75f3

Reply:
That commit has nothing to do with the variable being missing / undefined here :)